patches/minecraft/net/minecraft/block/BlockBubbleColumn.java.patch

```diff
@@ -0,0 +1,22 @@
+--- a/net/minecraft/block/BlockBubbleColumn.java
++++ b/net/minecraft/block/BlockBubbleColumn.java
+@@ -86,7 +86,7 @@
+       if (block == Blocks.field_203203_C) {
+          return iblockstate.func_177229_b(field_203160_a);
+       } else {
+-         return block != Blocks.field_150425_aM;
++         return p_203157_0_.func_180495_p(p_203157_1_.func_177977_b()).func_203425_a(net.minecraftforge.common.Tags.Blocks.BUBBLE_SOURCES_WITH_DRAG);
```

Comment:
I don't think this is equivalent. The original `block` comes from `iblockstate` which is at position `p_203157_1_`.
Why are you checking the block state below `p_203157_1_` here?